media/css/firefox/new/common/thanks-messaging.scss

```diff
@@ -71,14 +82,13 @@ $image-path: '/media/protocol/img';
     }
 
     &.linux {
-        .show-else,
-        .show-auto-download-notification {
+        .show-linux,
```

Comment:
Issue: we need to remember to update the `basic.html` template, as right now I get a blank page?

http://localhost:8000/en-US/firefox/download/thanks/?xv=basic

![image](https://github.com/mozilla/bedrock/assets/400117/a733154f-b917-4107-a7ba-9a25264233e8)